Breeding-Insight/bi-api
# [BI-2065] - Experiment overwrite missing phenotype validations

# Description
**Story:** [BI-2065 - Experiment overwrite does not do all data validations on phenotype data](https://breedinginsight.atlassian.net/browse/BI-2065)

Updated logic in ExperimentProcessor.java:validateObservations to run validations in the overwrite case

# Dependencies
n/a

# Testing

- Have one ontology term for each type of scale class
- Create experiment files, one with valid data for a phenotype corresponding to each scale class, and valid data for at least one timestamp, and one with invalid data for said phenotypes/timestamp
- Import first experiment file, should import successfully
- Import second experiment file, should get corresponding error messages for each incorrect phenotype and timestamp value

# Checklist:
